index.js

```diff
@@ -17,3 +18,96 @@ const menuDisappear = () => {
 };
 menuAppear();
 menuDisappear();
+
+// Popup window
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.